apps/web/components/atoms/checkbox/checkbox.tsx

```diff
@@ -0,0 +1,26 @@
+type CheckboxProps = {
+	id: string;
```

Comment:
Myślę, że zamiast przekazywać `id` jako props możnaby generować je za pomocą hooka `useId`. Ewentualnie props `id` dać jako opcjonalny i jeśli nie zostanie podany to generować go za pomocą `useId`.

Reply:
Faktycznie, nie ma potrzeby nadawania jakiegoś specjalnego id, to wygenerowane wystarczy